src/cogs/forums.py

```diff
@@ -60,6 +61,10 @@ async def tohelp(self, ctx: commands.Context):
 
         assert isinstance(ctx.channel, discord.TextChannel)
 
+        if not (permissions.is_staff(ctx.author, ctx.channel) \
+                or permissions.has_role(ctx.author, conf.helpful_role)):
```

Comment:
this is a little misleading, I'd prefer
`if not permissions.is_staff(ctx.author, ctx.channel) and not permissions.has_role(ctx.author, conf.helpful_role:`